src/components/Planters.js

```diff
@@ -126,62 +128,50 @@ const useStyles = makeStyles((theme) => ({
 const Planters = (props) => {
   // log.debug('render: Planters...');
   const classes = useStyles(props);
-  const [isFilterShown, setFilterShown] = React.useState(false);
-  const [isDetailShown, setDetailShown] = React.useState(false);
-  const [planterDetail, setPlanterDetail] = React.useState({});
+  const { planters, setPlanters } = useContext(AppContext);
+  const [isFilterShown, setFilterShown] = useState(false);
+  const [isDetailShown, setDetailShown] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [filter, setFilter] = useState(new FilterPlanter());
+  const [planterCount, setPlanterCount] = useState(0);
+  const [planterDetail, setPlanterDetail] = useState({});
+  const [pageSize, setPageSize] = useState(24);
+  const [pageNumber, setPageNumber] = useState(0);
```

Comment:
How about a object state:
{
  planterCount: 
  pageSize
  pageNumber
}

Because I suppose these props would be updated together, usually?

Reply:
Thanks for that idea!  I'll take a look and see if I can consolidate them.